ros2_control_demo_robot/description/diffbot_system.urdf.xacro

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+<!-- Revolute-Revolute Manipulator -->
+<!--
+Copied and modified from ROS1 example:
+https://github.com/ros-simulation/gazebo_ros_demos/blob/kinetic-devel/diffbot_description/urdf/diffbot.xacro
+-->
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="2dof_robot">
+
+  <!-- Import diffbot macro -->
+  <xacro:include filename="diffbot/diffbot.urdf.xacro" />
+  <!-- <xacro:include filename="$(find ros2_control_demo_robot)/description/diffbot/diffbot.urdf.xacro" /> -->
+
+  <!-- Import all Gazebo-customization elements, including Gazebo colors -->
+  <xacro:include filename="diffbot/diffbot.gazebo.xacro" />
+  <!-- <xacro:include filename="$(find ros2_control_demo_robot)/description/diffbot/diffbot.gazebo.xacro" /> -->
+
+  <!-- Import Rviz colors -->
+  <xacro:include filename="diffbot/diffbot.materials.xacro" />
+  <!-- <xacro:include filename="$(find ros2_control_demo_robot)/description/diffbot/diffbot.materials.xacro" /> -->
+
+  <!-- Import diffbot ros2_control description -->
+  <xacro:include filename="diffbot/diffbot_system.ros2_control.xacro" />
+  <!-- <xacro:include filename="$(find ros2_control_demo_robot)/description/diffbot/diffbot_system.ros2_control.xacro" /> -->
+
+  <!-- Used for fixing robot -->
+  <link name="world"/>
+  <gazebo reference="world">
+    <static>true</static>
+  </gazebo>
+
+  <xacro:diffbot parent="world" prefix="">
+    <origin xyz="0 0 0" rpy="0 0 0" />
```

Comment:
This does not make any sense with a mobile robot, since it is not fixed to the world. Or am I wrong?

Reply:
That's right.